### PR TITLE
[dagster-hightouch] standardize on integration project structure

### PR DESCRIPTION
**Pull request by cmpadden**
* Adopt project structure of existing integrations

### CODE REVIEW

**Comment by cmpadden** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21165?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#21165** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21165?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#21160** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21160?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21159** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21159?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21156** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21156?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @cmpadden and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python_modules/libraries/dagster-hightouch/integration.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Dagster Embedded ELT
```
Comment (tacastillo) [inline python_modules/libraries/dagster-hightouch/integration.yaml:1]: Heads up!

File: python_modules/libraries/dagster-hightouch/dagster_hightouch_tests/test_placeholder.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def test_placeholder():
```
Comment (tacastillo) [inline python_modules/libraries/dagster-hightouch/dagster_hightouch_tests/test_placeholder.py:1]: Is there a reason there is a placeholder test?

File: python_modules/libraries/dagster-hightouch/dagster_hightouch/__init__.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+from .ops import (
+    hightouch_sync_op as hightouch_sync_op,
+)
+from .resources import (
+    HightouchResource as HightouchResource,
+)
+from .version import __version__ as __version__
+
+DagsterLibraryRegistry.register("dagster-fivetran", __version__)
```
Comment (tacastillo) [inline python_modules/libraries/dagster-hightouch/dagster_hightouch/__init__.py:11]:
```suggestion
DagsterLibraryRegistry.register("dagster-hightouch", __version__)
```

File: python_modules/libraries/dagster-hightouch/dagster_hightouch/version.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__version__ = "1!0+dev"
```
Comment (tacastillo) [inline python_modules/libraries/dagster-hightouch/dagster_hightouch/version.py:1]: I spot checked a couple other libs but might've missed it: do our libs need a `version.py`?

**Review by tacastillo — CHANGES_REQUESTED** [review]
Small easy fixes by virtue of copy+pasting, which was the right thing to do lol. lmk

File: python_modules/libraries/dagster-hightouch/dagster_hightouch_tests/test_placeholder.py (hunk 2 above)
Comment (cmpadden) [reply python_modules/libraries/dagster-hightouch/dagster_hightouch_tests/test_placeholder.py:1]: Tox was failing with a strange error (exit status 5) when there were no tests present.  🤔

File: python_modules/libraries/dagster-hightouch/dagster_hightouch/__init__.py (hunk 3 above)
Comment (PedramNavid) [inline python_modules/libraries/dagster-hightouch/dagster_hightouch/__init__.py:11]: hightouch not fivetran :)

File: python_modules/libraries/dagster-hightouch/setup.cfg [hunk 5]
```diff
@@ -1,19 +1,8 @@
 [metadata]
-name = dagster-hightouch
-author = Pedram Navid
-author_email = pedram@hightouch.io
```
Comment (PedramNavid) [inline python_modules/libraries/dagster-hightouch/setup.cfg:4]: rip

**Review by PedramNavid — APPROVED** [review]
yay

File: python_modules/libraries/dagster-hightouch/dagster_hightouch/version.py (hunk 4 above)
Comment (tacastillo) [reply python_modules/libraries/dagster-hightouch/dagster_hightouch/version.py:1]: oh jk you're right. I just looked at the wrong dir

**Pull request merged**